### ISSUE TITLE
Conflicting LinkWith properties

### ISSUE THREAD

**Issue by breyed** [MicrosoftDocs/xamarin-docs#354]
In this example:

``` c#
[assembly: LinkWith ("libMagicChord.a", SmartLink = true, ForceLoad = true)]
```

Both `SmartLink` and `ForceLoad` are set. Yet, the `SmartLink` docs say this:
> The SmartLink property should be set to true to let Xamarin.iOS determine whether ForceLoad is required or not.

So why would you set both properties?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2dbcd55f-92cc-8859-7a9f-846b383e9f40
* Version Independent ID: d60d359b-007c-7731-be2d-92a49f106948
* Content: [Binding Objective-C libraries - Xamarin](https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/objective-c-libraries?context=xamarin%2Fios&tabs=vswin#feedback)
* Content Source: [docs/cross-platform/macios/binding/objective-c-libraries.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/cross-platform/macios/binding/objective-c-libraries.md)
* Service: **unspecified**
* Product: **xamarin**
* GitHub Login: @bradumbaugh
* Microsoft Alias: **brumbaug**

**Comment by bradumbaugh** [comment]
Hi @breyed - I'm looking in to this and will get back to you once I have some information. Thanks for calling this out!

**Comment by bradumbaugh** [comment]
@breyed – it turns out that both are present for legacy reasons. Older versions of MonoTouch didn't support SmartLink (so it would be ignored), but did need ForceLoad to be set to true. In newer versions of Xamarin.iOS, if SmartLoad is set to true, then ForceLoad is ignored. Setting both to true just covers all the bases. I'll find a way to update the docs to make this more clear. Thanks!

**Comment by breyed** [comment]
Makes sense. I would think all you need to do is update the code to correspond to a recent version of MonoTouch. Someone adding a new Binding Library project isn't going to be using an old MonoTouch version, right? In any case, are you want to close the issue without a linked commit?

**Comment by bradumbaugh** [comment]
Hi @breyed – sorry, I should have been clear that I've captured this on our internal backlog. There are a few places I'd like to update to make sure it's consistent across the board. I'm closing the issue as an indication that I've responded to the feedback and have captured the action.

I think you are likely OK to leave off the `ForceLoad=true` in a newly-created binding, but there is no harm in leaving it there.

Thanks again for your help!
